<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>panControlWidget</class>
 <widget class="QWidget" name="panControlWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="bSlider" name="positionSlider">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>45</number>
   </property>
  </widget>
  <widget class="bSlider" name="speedSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>550</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="coordLabel">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>11</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>234</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>bSlider</class>
   <extends>QSlider</extends>
   <header location="global">bslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>false</bool>
  </property>
  <property name="gridSnapY">
   <bool>false</bool>
  </property>
  <property name="gridVisible">
   <bool>false</bool>
  </property>
 </designerdata>
</ui>
